library/src/trips/types.d.ts

```diff
@@ -0,0 +1,18 @@
+declare module "all-the-cities" {
+  export type LngLat = [number, number];
+
+  export type City = {
+    cityId: Number;
+    name: string;
+    altName: string;
+    country: string;
+    featureCode: string;
+    adminCode: string;
+    population: number;
+    loc: { type: "Point"; coordinates: LngLat };
+  };
+
+  const Cities: City[];
+
+  export default Cities;
+}
```

Comment:
Why not use https://www.npmjs.com/package/@types/all-the-cities ?

Reply:
Oups I haven't seen it, updated 👍